src/bindings/java/org/simgrid/msg/Host.java

```diff
@@ -143,7 +143,13 @@ public boolean hasData() {
 	 * the value will be updated in kernel mode before returning the control to the requesting actor.
 	 */
 	public native double getConsumedEnergy();
-	
+
+	public native double getCurrentLoad();
```

Comment:
Please add a little javadoc chunk to these new calls

Reply:
done